testlib/test_plans/daily-daily-iso.plan.yaml

```diff
@@ -0,0 +1,19 @@
+name: daily daily-iso
+description: Daily run on current rawhide with anaconda dnf and python-blivet daily builds
+point_person: rvykydal@redhat.com
+artifact_type: github.scheduled.daily.kstest.daily-iso
+verified_by:
+  test_cases:
+    query: '"knownfailure" not in tc.tags and
+            "manual" not in tc.tags and
+            "skip-on-fedora" not in tc.tags and
+            "rhbz2018913" not in tc.tags and
+            "gh576" not in tc.tags and
+            "gh640" not in tc.tags and
+            "gh641" not in tc.tags and
+            "gh658" not in tc.tags and
+            "rhbz2052038" not in tc.tags and
+            "gh595" not in tc.tags'
```

Comment:
That is definitely something I need to figure out as a near step, depends on when and where we establish point of the truth. But to prevent two places of definition I should either generate this from our list or get rid of the list.